lib/rucio/common/utils.py

```diff
@@ -595,7 +595,7 @@ def register_surl_algorithm(surl_callable, name=None):
 register_surl_algorithm(construct_surl_BelleII, 'BelleII')
 
 
-def construct_surl(dsn, filename, naming_convention=None):
+def construct_surl(dsn, scope, filename, naming_convention=None):
```

Comment:
I thought about this a bit and it is not obvious _why_ the scope gets passed here. It does not seem to be used, so someone reading the code (me) could get confused and think, that this is some legacy unused parameter. I'd suggest a comment to clarify, that this parameter may be used in custom implémentations.

Reply:
What do you mean ? it's being passed to a function a couple of lines below

Reply:
Done